src/Cookie/CookieJar.php

```diff
@@ -58,26 +58,6 @@ public static function fromArray(array $cookies, $domain)
     }
 
     /**
-     * Quote the cookie value if it is not already quoted and it contains
-     * problematic characters.
-     *
-     * @param string $value Value that may or may not need to be quoted
-     *
-     * @return string
-     */
-    public static function getCookieValue($value)
```

Comment:
Instead of removing this public method, let's mark it as deprecated, remove the rest of the docblock other than the `@deprecated` annotation, and have it just return the value as-is. That way anyone calling this directly wont break.